developer-license.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
What about creationDate?

Reply:
Not needed in any way. See the verified dev page for example.

Reply:
Recently, when I've been updating older articles the creation date has been added automatically which might be a sign that it's good to have but oh well